CRM/Core/Smarty/plugins/modifier.print_array.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Smarty plugin
+ * @package Smarty
+ * @subpackage plugins
+ */
+
+/**
+ * Smarty print_array modifier plugin
+ *
+ * Type:     modifier<br>
+ * Name:     print_array<br>
+ * Purpose:  formats variable contents for display in the console
+ * @link http://smarty.php.net/manual/en/language.modifier.debug.print.var.php
```

Comment:
From this (now invalid) link it looks like maybe it used to ship with core

Reply:
I think it was actually a copy of the debug print var modifier

Reply:
Looks like it was added here https://github.com/civicrm/civicrm-svn/commit/a0b49959533df5a69379c8c504c060c5a702750e#diff-bab133e19b14acb008e3e5323e6078e7feee8998347d0de28ae3fdec1d3a7265R1 as part of https://issues.civicrm.org/jira/browse/CRM-8236

Reply:
@seamuslee001 OK - I see it used here

https://github.com/civicrm/civicrm-core/blob/8390b11ba7db0258ab7930526c4cb63929d8e0a3/tests/templates/documentFunction.tpl#L51

I'm OK with merging this but I want a bit more work on the comments so when people say 'can this be removed now' the comment block will make it clear they just need to grep '|@print_array}'

Reply:
@eileenmcnaughton thanks for that feedback, I have updated the comments now and also swapped out to use the CiviCRM copywrite as I cannot see it was ever part of the actual Smarty Package